pokemongo_bot/cell_workers/incubate_eggs.py

```diff
@@ -106,15 +106,13 @@ def _check_inventory(self, lookup_ids=[]):
                         "used": False
                     })
                 elif 'is_egg' not in pokemon and pokemon['id'] in lookup_ids:
-                    matched_pokemon.append({
-                        "pokemon_id": pokemon.get('pokemon_id', -1),
-                        "cp": pokemon.get('cp', 0),
+                    matched_pokemon.append(pokemon.update({
```

Comment:
Why does a PR that is adding a task for renaming pokemon need to modify the incubate eggs task?